### ISSUE TITLE
Keptn Full Tour on Prometheus v11 version issues

### ISSUE THREAD

**Issue by braghettos** [keptn/tutorials#210]
Hi everyone,

I'm following this tutorial: https://tutorials.keptn.sh/tutorials/keptn-full-tour-prometheus-011 but I think there are some issues regarding versions of repositories.

Let me describe the issues:

- STEP 3: this is the command suggested: `curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.11.2 sh` -
but Istio has been released in version 1.12.1. Which one should I use?

- STEP 4: this is the command suggested: `curl -sL https://get.keptn.sh | KEPTN_VERSION=0.10.0` bash but Keptn has been released in version 0.11.3. Which one should I use?

- STEP 6: this is the command suggested: `curl -o configure-istio.sh https://raw.githubusercontent.com/keptn/examples/0.10.0/istio-configuration/configure-istio.sh` but I can see there is the new release 0.11.0. Which one should I use?
- STEP 8: this is the command suggested: git clone --branch release-0.10.0 https://github.com/keptn/examples.git --single-branch but in the repo I can find the branch https://github.com/keptn/examples/tree/release-0.11.0. Which one should I use?

Thanks for any feedback!

**Comment by oleg-nenashev** [comment]
Thanks for the link @braghettos! I will be updating the tutorials layout so that the versions inside the tutorial can be maintained in a centralized way. Unfortunately we do not use Asciidoc so it will require some hacks....

* Regarding the versions, I suggest using the latest versions of everything for now, We will need to update them anyway
* For https://github.com/keptn/examples.git, use the master branch for now. There were some updates that currently do not present in version tags. The branching model seems to be deprecated entirely and replaced by tags. CC @johannes-b @christian-kreuzberger-dtx to confirm

**Comment by christian-kreuzberger-dtx** [comment]
The problems mentioned are not related to Prometheus, but to Keptn Core. FYI @thisthat

**Comment by thisthat** [comment]
Yes, I confirm that the release of https://github.com/keptn/examples is based on tags and not branching.

**Comment by braghettos** [comment]
@thisthat regarding step 3 and 4 I don't see any changes in https://deploy-preview-211--keptn-tutorials.netlify.app/tutorials/keptn-full-tour-prometheus-011/index.html?index=..%2F..keptn011x#2 and https://deploy-preview-211--keptn-tutorials.netlify.app/tutorials/keptn-full-tour-prometheus-011/index.html?index=..%2F..keptn011x#3

**Comment by braghettos** [comment]
In https://deploy-preview-211--keptn-tutorials.netlify.app/tutorials/keptn-full-tour-prometheus-011/index.html?index=..%2F..keptn011x# I would expect to use Istion v1.12.1

**Comment by braghettos** [comment]
In https://deploy-preview-211--keptn-tutorials.netlify.app/tutorials/keptn-full-tour-prometheus-011/index.html?index=..%2F..keptn011x#3 I would expect to use Keptn v0.11.3

**Comment by thisthat** [comment]
I failed with my sed skils 🙈 Thank you @braghettos, I'll provide a follow up PR

**Comment by braghettos** [comment]
@thisthat still in https://deploy-preview-212--keptn-tutorials.netlify.app/tutorials/keptn-full-tour-prometheus-011/index.html?index=..%2F..index#2 there is an issue:

`./istio-1.11.2/bin/istioctl install`

should be

`./istio-1.12.1/bin/istioctl install`
